### ISSUE TITLE
Tasks are not visible after adding

### ISSUE THREAD

**Issue by FireRacing** [iman-salmani/iplan#63]
After I add any tasks or subtasks, I am not able to view them. After adding, they end up like this:

![Screenshot_2023-07-23_11-48-47](https://github.com/iman-salmani/iplan/assets/33418792/09f116b4-a23f-443f-a4c1-c2bea5fa8650)

**Comment by iman-salmani** [comment]
Hi @FireRacing 
Depending on your screenshot I assume you were using Ubuntu and maybe the Snap version which is a few path versions behind (in the stable channel).
I'm in touch with the maintainer of the Snap version and hopefully, we will find a way to verify and keep it up-to-date.
For now, I suggest using the flatpak version in [flathub](https://flathub.org/apps/ir.imansalmani.IPlan).

If you were using version 1.9.2:
Do you mean after pressing add task button the name entry of created task does not grab focus?
Or you edit them but won't apply in UI. If so, What happens when you restart the app?

**Comment by FireRacing** [comment]
I used the flatpak version and I didn't use Ubuntu. Instead it was a version of Arch Linux called EndeavourOS with XFCE that bases itself on Arch Linux with an extra repository of it's own. 

For your first question, I set them by pressing enter after typing in the task.

For your second question, An edit option is absent in the UI.

![Screenshot_2023-07-24_19-34-21](https://github.com/iman-salmani/iplan/assets/33418792/8a170eed-2da2-46bc-bcbd-9d0da82b1da4)

The UI remained like this after restarting

![Screenshot_2023-07-24_19-38-18](https://github.com/iman-salmani/iplan/assets/33418792/81903e2f-8d52-4516-904c-9233ce8d5596)

This is the output of inxi -G:

Graphics:
  Device-1: NVIDIA TU116 [GeForce GTX 1660] driver: nvidia v: 535.86.05
  Display: x11 server: X.Org v: 21.1.8 driver: X: loaded: nvidia
    unloaded: modesetting gpu: nvidia,nvidia-nvswitch resolution: 1920x1080~60Hz
  API: OpenGL v: 4.6.0 NVIDIA 535.86.05 renderer: NVIDIA GeForce GTX
    1660/PCIe/SSE2

For your information, I use darkman to automate transitioning of light and dark modes system wide or if I want to set light or dark mode myself system wide.
